#! /bin/sh /usr/share/dpatch/dpatch-run
## 13armfix.dpatch by  <bernat@neo.luffy.cx>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix bus-error on ARM, thanks to Sven-Ola Tuecke.

@DPATCH@

--- xrdp-0.4.0~dfsg/common/arch.h.orig	2009-08-31 07:56:08.000000000 +0200
+++ xrdp-0.4.0~dfsg/common/arch.h	2009-08-31 07:57:05.000000000 +0200
@@ -35,7 +35,7 @@
 /* check if we need to align data */
 #if defined(__sparc__) || defined(__alpha__) || defined(__hppa__) || \
     defined(__AIX__) || defined(__PPC__) || defined(__mips__) || \
-    defined(__ia64__) || defined(__ppc__)
+    defined(__ia64__) || defined(__ppc__) || defined(__arm__)
 #define NEED_ALIGN
 #endif
 
